src/components/QuestCard.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Card } from "semantic-ui-react";
+
+const QuestCard = (props) => {
+  return (
+    <>
+      <Card id={"quest-" + props.quest.id}>
+        <Card.Content>
+          <Card.Header>{props.quest.title}</Card.Header>
+          <Card.Meta>{props.quest.kpoints} KP</Card.Meta>
```

Comment:
Small note: I think it's a bit cleaner to take deconstructed props like ({ quest }), then we can say `quest.id` instead of `props.quest.id` and so on

Reply:
done